src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/CngKey.Create.cs

```diff
@@ -149,6 +159,114 @@ private static void InitializeKeyUiPolicyProperties(SafeNCryptKeyHandle keyHandl
                 }
             }
         }
+
+        /// <summary>
+        ///     Return a CngProperty representing a named curve.
+        /// </summary>
+        internal static CngProperty GetPropertyFromNamedCurve(ECCurve curve)
```

Comment:
Yeah I was thinking that too. Changed it to your recommendation. Originally thinking it was more "create" focused than "ec" and it seemed like overkill to create a new key.ec.create...